components/searchbar/no-results.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+
+const NoResultsModal = ({ clickHandler, noResultsModal }) => {
+  return (
+    <div>
+      {!noResultsModal ? null : (
```

Comment:
Same here. Use a ternary to pick between _two things_; not _one thing_ and _nothing_.